contents/docs/data-warehouse/views.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Creating views
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+The PostHog data warehouse supports views. With views, you can save your most used queries and reference them in subsequent queries.
+
+### Creating a view
+
+Query views are created directly inside the HogQL editor. If the query has valid view characteristics, the "Save as view" button will be enabled. You will be prompted to give the view a name which will be the name that can be refeneced in other queries. For a query to be a valid view, all fields being accessed must be aliased. The alias names will be how you access the fields on the view. 
+
+> **Note:** Currently, nested views are not supported. You cannot create a view based on other views only base PostHog models.
+
+![valid view](../../images/features/data-warehouse/valid-view.png)
```

Comment:
Since we're using the product usage data example below, it might be nice to have a query generating that as a view in this image! and then you can show the view being selected in the "Link view to table" image below.

Reply:
Done!